ern-local-cli/src/commands/transform-container.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  utils as coreUtils,
+  Platform,
+  NativePlatform,
+  fileUtils,
+} from 'ern-core'
+import { transformContainer } from 'ern-container-transformer'
+import utils from '../lib/utils'
+import fs from 'fs'
+import path from 'path'
+import { Argv } from 'yargs'
+
+export const command = 'transform-container'
+export const desc = 'Transform a local Container'
+
+export const builder = (argv: Argv) => {
+  return argv
+    .option('containerPath', {
+      describe: 'Local path to the Container to transform',
+      type: 'string',
+    })
+    .option('config', {
+      alias: 'c',
+      describe:
+        'Optional transformer configuration (json string or path to config file)',
+      type: 'string',
+    })
+    .option('platform', {
+      choices: ['android', 'ios'],
+      demandOption: true,
+      describe: 'Native platform of the Container',
+      type: 'string',
+    })
+    .option('transformer', {
+      alias: 't',
+      demandOption: true,
+      describe: 'Transformer to use',
+      type: 'string',
+    })
+    .epilog(utils.epilog(exports))
+}
+
+export const handler = async ({
+  containerPath,
+  config,
+  platform,
+  transformer,
+}: {
+  containerPath?: string
+  config?: string
+  platform: NativePlatform
+  transformer: string
```

Comment:
throw error if `transformer` is null or undefined

Reply:
It cannot be null or undefined because we set the `yargs` option `demandOption: true`.
With that option set, it is not possible for the user not to provide `-transformer/--t`. The command will fail with error thrown by `yargs` : `Missing required arguments:  transformer`